.github/workflows/build-wheels.yml

```diff
@@ -0,0 +1,70 @@
+name: build-wheels
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - '*'
+
+concurrency:
+  group: build-wheels-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest]
```

Comment:
Is macOS not supported?

Reply:
I thought macOS belongs to manylinux, will add macOS, thanks!